autogen/browser_utils/mdconvert.py

```diff
@@ -0,0 +1,1046 @@
+# ruff: noqa: E722
+import os
+import re
+import io
+import sys
+import requests
+import json
+import markdownify
+
+# File-format detection
+import puremagic
+import mimetypes
+from binaryornot.check import is_binary
```

Comment:
Should we include pptx, binaryornot, puremagic, mammoth, etc. into the `setup.py`? If they are not in the existing dependency... I did not find them in the websurfer option.